heka/sandbox/decoders/extract_executive_summary.lua

```diff
@@ -211,26 +212,28 @@ function process_message()
 
     if doc_type == "main" then
         set_string_field(msg.Fields.reason, "Fields[reason]")
-        msg.Fields.hours.value              = 0
-        msg.Fields.google.value             = 0
-        msg.Fields.bing.value               = 0
-        msg.Fields.yahoo.value              = 0
-        msg.Fields.other.value              = 0
-        msg.Fields.sessionId.value          = ""
-        msg.Fields.subsessionCounter.value  = 0
-        msg.Fields.pluginHangs.value        = 0
+        msg.Fields.hours.value               = 0
```

Comment:
nit: some of this alignment whitespace can be removed (also above in "crash_fields" and "main_fields").